app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialRepositoryImpl.java

```diff
@@ -343,7 +343,7 @@ private FieldUiModel transform(@NonNull ProgramStageDataElement stage, DataEleme
                 optionSetConfig,
                 null,
                 null,
-                null);
+                null, null);
     }
```

Comment:
is this a formatting typo?

Reply:
the new null param is correct but I'm not sure why KtlintFormat didn't catch that, I'll update it